src/kata6.joinNames.js

```diff
@@ -1,3 +1,17 @@
-const joinNames = (namesObj) => {};
+/* eslint-disable no-plusplus */
+const joinNames = (namesObj) => {
+    const namesArr = namesObj.map(person => person.name);
+    let nameStr = '';
+    namesArr.forEach((name, index) => {
+        if(index === namesArr.length - 2) {
+           nameStr += name + ' & ';
+        } else if (index === namesArr.length - 1) {
+           nameStr += name + '';
```

Comment:
The adding of the zero length string doesnt make any sense here, you can remove this
```
           nameStr += name;
```